scripts/helper/create-topic.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+config=$1
+topic=$2
+
+export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
+   --bootstrap-server kafka1:11091 \
+   --command-config /etc/kafka/secrets/$config \
+   --topic $topic \
+   --create \
+   --replication-factor 2 \
+   --partitions 2 \
+   --config confluent.value.schema.validation=true
```

Comment:
This does change cp-demo behavior in that now ALL these topics have `confluent.value.schema.validation=true`.  It's worth noting this change and thinking through potential impact.

Reply:
Thanks - I did actually notice this when refactoring, but can't remember why I didn't address it ... perhaps I thought it should be safe, or forgot ... I'll check what we should do here.